apps/web-giddh/src/app/userDetails/components/subscription/subscription.component.html

```diff
@@ -5,8 +5,8 @@
                 {{ commonLocaleData?.app_buy_new_plan }}
             </button>
         </div>
-        <mat-grid-list cols="16" rowHeight="40px" gutterSize="0px" class="responsive-grid-list">
-            <mat-grid-tile colspan="4" rowspan="1" class="responsive-grid-title-left-top">
+        <mat-grid-list cols="16" rowHeight="40px" gutterSize="0px">
+            <mat-grid-tile colspan="{{ menuOneWidth }}" rowspan="1">
```

Comment:
Consider using `[attr.colspan]="menuOneWidth"` through out this file where colspan is used